packages/apps-engine/src/server/AppManager.ts

```diff
@@ -492,6 +496,37 @@ export class AppManager {
         return true;
     }
 
+    public async migrate(id: string): Promise<boolean> {
+        const app = this.apps.get(id);
+
+        if (!app) {
+            throw new Error(`No App by the id "${id}" exists.`);
+        }
+
+        await app.call(AppMethod.ONUPDATE).catch((e) => console.warn('Error while migrating:', e));
```

Comment:
I had to trigger an update here, otherwise the migration wouldn't be "visible" in the client side after a workspace is downgrade. Not sure if there's another way to do so without having to trigger an update, but I couldn't find any other way by the time I implemented this